src/styles/blocks/card.scss

```
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  @extend %card-border;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+  margin: 0 48px 46px 0;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__title,
+  &__rating,
+  &__price,
+  &__pay {
+    margin: #{$global-margin};
+  }
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__title {
+    & :first-child {
+      margin: 0;
+      @extend %product-name-text;
+      color: #{$main-color};
+    }
+
+    & p:last-child {
+      margin: 4px 0 0;
+      @extend %product-code-text;
+      color: #{$secondary-color};
+      margin-bottom: 8px;
+    }
+
+  }
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 1px;
+
+    &_feedback {
+      font-weight: 400;
+      font-size: 10px;
+    }
+  }
```

`feedback` should be an element of `card` block.
```suggestion
  &__rating {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-bottom: 1px;
  }
  
  &__feedback {
      font-weight: 400;
      font-size: 10px;
    }
```